Signal/translations/en.lproj/Localizable.strings

```diff
@@ -70,6 +70,12 @@
 /* No comment provided by engineer. */
 "AUDIO_PERMISSION_MESSAGE" = "Signal requires access to your microphone to work properly. You can grant this permission in the Settings app >> Privacy >> Microphone >> Signal";
 
+/* Title for call interstitial view */
+"CALL_INTERSTITIAL_CALLING_LABEL" = "Calling...";
+
+/* Label for cancel button on call interstitial view */
+"CALL_INTERSTITIAL_CANCEL_BUTTON" = "Cancel";
```

Comment:
Care to use our existing cancel text, or does this deserve it's own translation?